README.md

```diff
@@ -573,441 +468,19 @@ debugger attaches, and you can then step through the code.
-./emissary feed -i ~/Desktop/feed1/
-```
-
-You should be able to hit http://localhost:8001, http://localhost:9001 and http://localhost:7001 in the browser and
-look at the configured places.  Drop some files in the ~/Desktop/feed1 and see the 2 nodes process them.  It may 
-take a minute for them to start processing
-
-### Agents (Cluster)
-
-Agents in clustered mode again shows details about the mobileAgents.  It starts at with the node you 
-configure (localhost:9001 by default), then calls out to all nodes it knows about and gets the same 
-information.  Run it like so:
-
-```
-./emissary agents --cluster
-```
-
-### Pool (Cluster)
-
-Pool in clustered mode also does the same pool in standalone.  It starts at the node (locahost:9001) by default
-then goes to all the nodes it knows about and aggregates a collapsed view of the cluster.  Run it with
-
-```
-./emissary pool --cluster
-```
-
-### Topology (Clustered)
-
-The topology talks to the configured node (localhost:8001 by default) and talks to every it knows about.
-The response is what all those nodes know about, so you can build up a network topology of your cluster.
-Run it with 
-
-```
-./emissary topology
-```
-
-### Running server with SSL
-
-The keystore and keystore password are in the emissary.client.EmissaryClient-SSL.cfg file.  Included and configured
-by default is a sample keystore you can use for testing this functionality.  It is not recommended to use in 
-production environments.  To use your own keystore, change configuration values in the 
-emissary.client.EmissaryClient-SSL.cfg file.
-
-Standalone
-
-```
-./emissary server -p 8443 --ssl
-```
-
-Clustered
-```
-./emissary server -p 8443 --ssl --mode cluster
-./emissary server -p 9443 --ssl --mode cluster
-mkdir ~/Desktop/feed1
-./emissary feed -p 7443 --ssl -i ~/Desktop/feed1/
-````
-
-### Debugging a command
-
-The _emissary_ script allows you to remote debug any command.  Simply use `DEBUG=true` before the command and it will 
-wait for a remote debugger to attach at port 8000.  If you want to change that port, use `DEBUG_PORT` instead.  
-For example:
-
-```
-DEBUG=true ./emissary <command> <command_options>
-```
-
-or 
-
-```
-DEBUG_PORT=<someport> ./emissary <command> <command_options>
-```
-
-You can also debug what the script is doing by prepending the following
-
-```
-DEBUG_CMD=true
-```
-
-This will output the command that is going to be run and show you how JCommander is parsing the arguments.
-
-#### Setting up remote debugging in your IDE
-
-For info on how to setup remote debugging in your IDE, reference the
-following articles:
-
-- for
-[Eclipse](http://help.eclipse.org/kepler/index.jsp?topic=%2Forg.eclipse.jubula.client.ua.help%2Fhtml%2Freference%2Fnode47.html)
-- for [IntelliJ](https://www.jetbrains.com/idea/help/run-debug-configuration-remote.html)
-
 ## Docker
 
-Docker needs to be installed locally to build images. If developing on Linux, the maven plugin for Docker cannot be run 
```

Comment:
Moved all of the Docker info into a separate README so that it is self-contained and easy to find